src/Files.App/DataModels/NavigationControlItems/FileTagItem.cs

```diff
@@ -19,16 +19,16 @@ public string Path
 			}
 		}
 
-		public string ToolTipText { get; private set; }
+		public string? ToolTipText { get; private set; }
 
-		public SectionType Section { get; set; }
+		public SectionType? Section { get; set; }
 
-		public ContextMenuOptions MenuOptions { get; set; }
+		public ContextMenuOptions? MenuOptions { get; set; }
 
-		public NavigationControlItemType ItemType => NavigationControlItemType.FileTag;
+		public NavigationControlItemType? ItemType => NavigationControlItemType.FileTag;
 
-		public int CompareTo(INavigationControlItem other) => Text.CompareTo(other.Text);
+		public int CompareTo(INavigationControlItem? other) => (Text is not null ? Text.CompareTo(other?.Text) : 0);
```

Comment:
Writing two lines instead of one here would prevent a double ternary operator, which are a bit of a pain for the clarity of the code.

Reply:
Is it not only one ternary operator?